config/routes.rb

```diff
@@ -10,4 +10,9 @@
     resources :comments
   end
   resources :assigned_routes
+  resources :maintenance_requests do
+    member do
```

Comment:
from what I understand, this allows me to declare custom actions within a resource. So I just added one for resolving a maintenance request.